privateSBOMExchange/src/petra/lib/models/__init__.py

```diff
@@ -1,41 +1,319 @@
-from smt.tree import TreeMapStore, TreeMemoryStore
+import configparser
+from typing import List
+import re
+
+from smt.tree import TreeMemoryStore
 from smt.tree import SparseMerkleTree
 from smt.utils import DEFAULTVALUE, PLACEHOLDER
 from smt.proof import verify_proof
 from smt.tree import SparseMerkleTree
 
-from typing import Tuple
-import json
+import hashlib
+from lib4sbom.parser import SBOMParser
+
+from cpabe import cpabe_encrypt,cpabe_setup,cpabe_keygen,cpabe_decrypt
+
+pk, mk = cpabe_setup()
+GROUPS = ["FEDRAMP" , "COMPLIANCE"]
```

Comment:
Not sure if this initialization code should live here, or whether we should hardcode the groups here either.
I'll leave it for now, but we should turn this into a ticket (to provide an interface to initialize an SBOM tree with a keypair)

Reply:
just added it